input/pagecontent/developerGuide.md

```diff
@@ -18,13 +18,78 @@ In order to make requests against this API clients must:
     "use": "official",
     "system": "https://standards.digital.health.nz/ns/hpi-person-id",
     "value": "99ZZFX"
-  }
+  },
+  "encryptedClaims": [],
+  "encryptedKid": "01"
 }
 ```
 
 <p>&nbsp;</p>
 
-# Well Known Resource Identifiers
+## Well Known Resource Identifiers
 
 
 Some resources (Plan Definitions and Questionnaires) are defined by Te Whatu Ora and can be searched for using their well-known identifiers.  Please refer to the example `identifier` parameters that can be supplied to the `GET /PlanDefinition` and `GET /Questionnaire` operations for the current set of well-known identifiers.
+
+
+## Field level request encryption
+
+The FHIR server supports certain FHIR resource fields to be provided in the create or update request in an encrypted format, to prevent certain data such as PII being transmitted in plain text.
+
+The currently supported fields for this feature are:
+- "subject.identifier.value"
```

Comment:
Did you want to add `subject.display` as it is encrypted in the sequence diagram? I wonder if we can make it `*.identifier.value`?

Reply:
As discussed, the wildcard feature would need some additional thought but could be added later. The subject.display is included in this list (which can easily be expanded)